bitbake/lib/bb/fetch2/repo.py

```diff
@@ -82,7 +82,7 @@ def download(self, ud, d):
             tar_flags = "--exclude='.repo' --exclude='.git'"
 
         # Create a cache
-        runfetchcmd("tar %s -czf %s %s" % (tar_flags, ud.localpath, os.path.join(".", "*") ), d, workdir=ud.codir)
+        runfetchcmd("tar %s -cf - %s | pigz %s" % (tar_flags, os.path.join(".", "*"), ud.localpath), d, workdir=ud.codir)
```

Comment:
Copy-paste bug (my bad): output redirection from pigz to file is needed
runfetchcmd("tar %s -cf - %s | pigz > %s" % (tar_flags, os.path.join(".", "*"), ud.localpath), d, workdir=ud.codir)